config/notifiers.go

```diff
@@ -306,6 +317,37 @@ func (c *OpsGenieConfig) UnmarshalYAML(unmarshal func(interface{}) error) error
 	return checkOverflow(c.XXX, "opsgenie config")
 }
 
+// VictorOpsConfig configures notifications via VictorOps.
+type VictorOpsConfig struct {
+	NotifierConfig `yaml:",inline"`
+
+	APIKey       Secret `yaml:"api_key"`
+	APIURL       string `yaml:"api_url"`
+	RoutingKey   string `yaml:"routing_key"`
+	MessageType  string `yaml:"message_type"`
+	EntityID     string `yaml:"entity_id"`
```

Comment:
From my understand in the [API docs](http://victorops.force.com/knowledgebase/articles/Integration/Alert-Ingestion-API-Documentation/), Entity Id is very much like the OpsGenie "Source," there we use the __alertmanagerURL but in this case we're using Group labels, which may be more readable and understandablet oo.